Dockerfile

```diff
@@ -6,9 +6,14 @@ RUN gradle build
 
 #FROM amazoncorretto:22-alpine
 FROM openjdk:8-jre-slim
-
+EXPOSE 8080
```

Comment:
Remove this, we aren't exposing the docker port, everything will be on the internal docker network. If you want to use it for testing and still want to expose this port, use docker-compose to start your stack